prow/test/integration/test/horologium_test.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"context"
+	"testing"
+
+	"k8s.io/apimachinery/pkg/labels"
+	prowjobv1 "k8s.io/test-infra/prow/apis/prowjobs/v1"
+	"k8s.io/test-infra/prow/kube"
+	ctrlruntimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func TestLaunchProwJob(t *testing.T) {
+	const existJobName = "horologium-schedule-test-job"
+	t.Parallel()
+
+	tests := []struct {
+		name string
+	}{
+		{
+			name: "periodic-job-must-run",
+		},
+	}
+
+	for _, tt := range tests {
+		tt := tt
+		t.Run(tt.name, func(t *testing.T) {
+			t.Parallel()
+
+			clusterContext := getClusterContext()
+			t.Logf("Creating client for cluster: %s", clusterContext)
+			kubeClient, err := NewClients("", clusterContext)
+			if err != nil {
+				t.Fatalf("Failed creating clients for cluster %q: %v", clusterContext, err)
+			}
+			ctx := context.Background()
+
+			pjs := &prowjobv1.ProwJobList{}
+			err = kubeClient.List(ctx, pjs, &ctrlruntimeclient.ListOptions{
```

Comment:
Is it safe to assume that horologium always finished starting before we run this test?

Reply:
Good point, for prow this probably is true, as it pushes test images before running tests https://github.com/kubernetes/test-infra/blob/b15f63b0d8faba951d40b6f11b9acf2d42934be5/prow/test/integration/setup-cluster.sh#L142. But I think it's better to wait for 1 minute just to be safe

Reply:
Done